README.md
[![Build Status](https://travis-ci.org/DanBennettUK/djriko.svg?branch=master)](https://travis-ci.org/DanBennettUK/djriko)

# DJRIKO / Producer / DJ

![Logo](/imgs/logo/logo.png)


releases.md
---
layout: default
permalink: /releases/
target: https://linktr.ee/ContagiousRecords
---


album1.php
<?php
/*
Created by Dan Bennett December 2013
For DJ Riko.
24th December 2013 - Merry Christmas & Happy new year!
*/
?>

<html>
	<head>
		<title>RIKO - CONTAGIOUS FREE ALBUM! DOWNLOAD NOW!</title>
		<link rel="image_src" href="/contagiousfront.jpg" / ><!--formatted-->
		<meta name="description" content="RIKO - CONTAGIOUS FREE ALBUM! DOWNLOAD NOW! 15 Unmixed tracks for download + Continuous mix!" /><!--formatted-->
	</head>
	<body style="background-color:black;">
	<a href="/" style="color:#fff; font-size:12pt; font-family:HelveticaNeue-Light,arial">< Back</a>
		<center>
			<img src="/contagiousfront.jpg" style="align:center;">
			<br />
			<img src="/contagiousback.jpg" style="align:center;">
			<p>
			<a href="/riko-contagious.zip" style="color:#fff; font-size:78pt; font-family:HelveticaNeue-Light,arial">Download Now!</a>
			<p>
			<a href="https://www.facebook.com/contagiousrecords" target="_blank"><img src="/facebook.png" height="42px" width="42px"></a><font color="black"> - - - </font><a href="https://soundcloud.com/contagious-records" target="_blank"><img src="/soundcloud.png" height="42px" width="42px"></a>
		</center>
                <font color="white">
			<?php
/*
                        $filePath = 'contagious1.txt';

                        // If file exists, read current count from it, otherwise, initialize it to 0
                        $count = file_get_contents($filePath);

                        // Display current download count
                        echo "Downloads: " . $count;
*/
                        ?>
		</font>
	</body>
</html>


album2.php
<?php
/*
Created by Dan Bennett December 2013
For DJ Riko.
24th December 2013 - Merry Christmas & Happy new year!
*/
?>

<html>
	<head>
		<title>RIKO - CONTAGIOUS FREE ALBUM! DOWNLOAD NOW!</title>
		<link rel="image_src" href="/contagiousfront.jpg" / ><!--formatted-->
		<meta name="description" content="RIKO - CONTAGIOUS FREE ALBUM! DOWNLOAD NOW! 15 Unmixed tracks for download + Continuous mix!" /><!--formatted-->
	</head>
	<body style="background-color:black;">
	<a href="/" style="color:#fff; font-size:12pt; font-family:HelveticaNeue-Light,arial">< Back</a>
		<center>
			<img src="/contagious2front.jpg" style="align:center;">
			<br />
			<img src="/contagious2back.jpg" style="align:center;">
			<p>
			<a href="/riko-contagious2.zip" style="color:#fff; font-size:78pt; font-family:HelveticaNeue-Light,arial">Download Now!</a>
			<p>
			<a href="https://www.facebook.com/contagiousrecords" target="_blank"><img src="/facebook.png" height="42px" width="42px"></a><font color="black"> - - - </font><a href="https://soundcloud.com/contagious-records" target="_blank"><img src="/soundcloud.png" height="42px" width="42px"></a>
		</center>
		<font color="white">
			<?php/*

                        $filePath = 'contagious2.txt';

                        // If file exists, read current count from it, otherwise, initialize it to 0
                        $count2 = file_get_contents($filePath);

                        // Display current download count
                        echo "Downloads: " . $count2;
*/
                        ?>
		</font>
	</body>
</html>


download.php
<?php
if (empty($_GET)) {
    echo "Nothing to download?";
    header("index.php");
    die();
}

$down=$_GET['down'];

?>


<?php

if ($down === "riko-contagious.zip") {

    ?>
    <html>
        <head>
            <meta http-equiv="refresh" content="0;url=<?php echo $down; ?>">
        </head>
        <body>
            <?php
            $filePath = 'contagious1.txt';

            // If file exists, read current count from it, otherwise, initialize it to 0
            $count = file_exists($filePath) ? file_get_contents($filePath) : 430;

            // Increment the count and overwrite the file, writing the new value
            file_put_contents($filePath, ++$count);

            // Display current download count
//            echo "Downloads:" . $count;
//            header("Location: index.php");

            ?>
        </body>
    </html>
    <?php
} elseif ($down === "riko-contagious2.zip") {
    ?>
    <html>
        <head>
            <meta http-equiv="refresh" content="0;url=<?php echo $down; ?>">
        </head>
        <body>
            <?php
            $filePath = 'contagious2.txt';

            // If file exists, read current count from it, otherwise, initialize it to 0
            $count2 = file_exists($filePath) ? file_get_contents($filePath) : 895;

            // Increment the count and overwrite the file, writing the new value
            file_put_contents($filePath, ++$count2);

            // Display current download count
//            echo "Downloads:" . $count2;
//
//            header("Location: index.php");
            ?>
        </body>
    </html>
<?php
} elseif ($down === "RIKO+POWER.zip") {
    ?>
    <html>
        <head>
            <meta http-equiv="refresh" content="0;url=<?php echo $down; ?>">
        </head>
        <body>
            <?php
            $filePath = 'rikopower.txt';

            // If file exists, read current count from it, otherwise, initialize it to 0
            $count2 = file_exists($filePath) ? file_get_contents($filePath) : 895;

            // Increment the count and overwrite the file, writing the new value
            file_put_contents($filePath, ++$count2);

            // Display current download count
//            echo "Downloads:" . $count2;
//
//            header("Location: index.php");
            ?>
        </body>
    </html>
<?php
}


rikopower.php
<?php
/*
Created by Dan Bennett July 2016
For DJ Riko.
*/
?>

<html>
	<head>
		<title>RIKO POWER - FREE ALBUM! DOWNLOAD NOW!</title>
		<link rel="image_src" href="/riko-power.png" / ><!--formatted-->
		<meta name="description" content="RIKO - 'RIKO POWER' FREE ALBUM! DOWNLOAD NOW!" /><!--formatted-->
	</head>
	<body style="background-color:black;">
	<a href="/" style="color:#fff; font-size:12pt; font-family:HelveticaNeue-Light,arial">< Back</a>
		<center>
			<img src="/riko-power.png" style="align:center; min-width:50%; max-width:75%;">
			<p>
			<a href="/RIKO+POWER.zip" style="color:#fff; font-size:78pt; font-family:HelveticaNeue-Light,arial">Download Now!</a>
			<p>
			<a href="https://www.facebook.com/contagiousrecords" target="_blank"><img src="/facebook.png" height="42px" width="42px"></a><font color="black"> - - - </font><a href="https://soundcloud.com/contagious-records" target="_blank"><img src="/soundcloud.png" height="42px" width="42px"></a>
		</center>
                <font color="white">
			<?php
/*
                        $filePath = 'rikopower.txt';

                        // If file exists, read current count from it, otherwise, initialize it to 0
                        $count = file_get_contents($filePath);

                        // Display current download count
                        echo "Downloads: " . $count;
*/
                        ?>
		</font>
	</body>
</html>


hellershardcore.php
<?php
/*
Created by Dan Bennett December 2013
For DJ Riko.
24th December 2013 - Merry Christmas & Happy new year!
*/
?>

<html>
	<head>
<title>CONTAGIOUS RECORDS - HELLER'S HARDCORE (OUT NOW!) - 12 Unmixed Exclusive Tracks</title>
                <link rel="image_src" href="/hhfront.jpg" / ><!--formatted-->
                <meta name="description" content="Hellers Hardcore (Out Now!) 12 Unmixed exclusive tracks!" /><!--formatted-->	
</head>
	<body style="background-color:black;">
	<a href="/" style="color:#fff; font-size:12pt; font-family:HelveticaNeue-Light,arial">< Back</a>
		<center>
			<img src="/hhfront.jpg" style="align:center; width:45%;">

			<img src="/hhback.jpg" style="align:center; width:45%;">
			<p>
		<h2 style="color:#fff; font-family:HelveticaNeue-Light,arial; text-align:center;">
		Contagious Records present..</h1>
		<h1 style="color:#fff; font-family:HelveticaNeue-Light,arial; text-align:center;">★ HELLER'S HARDCORE (1XCD) ★</h1>
		<h3 style="color:#fff; font-family:HelveticaNeue-Light,arial; text-align:center;">
		12 unmixed DJ friendly tracks from - Steve Heller, DJ RIKO, Joey Riot, Kurt, M-Project, Jonjo, Alex BassJunkie, Eufeion + More ..<br />
		<p>
		Buy Heller's Hardcore now for £6 + P&P! LIMITED STOCK LEFT!</h3>
		<p>
        <h2 style="color:#fff; font-family:HelveticaNeue-Light,arial; text-align:center;">Buy now on <a href="https://www.beatport.com/release/hellers-hardcore/1726082" style="color:#fff;">Beatport</a></h2>
		<p>
			<a href="https://www.facebook.com/contagiousrecords" target="_blank"><img src="/facebook.png" height="42px" width="42px"></a><font color="black"> - - - </font><a href="https://soundcloud.com/contagious-records" target="_blank"><img src="/soundcloud.png" height="42px" width="42px"></a>
		</center>
	</body>
</html>


index.php
<?php
/*
Created by Dan Bennett December 2013
For DJ Riko.
24th December 2013 - Merry Christmas & Happy new year!

UPDATED 4th March 2018. Added Unleashed album information.

*/
?>

<html>
<head>
    <title>Contagious Records Albums, Free Music + More</title>
    <link rel="image_src" href="/unleashed_front.jpg" / >
    <meta name="description" content="Contagious Records Albums, Free Music + More" />
    <link rel="stylesheet" type"text/css" href="/style.css">
</head>
<body style="background-color:black;">
    <div class="header" style="text-align: center;">
        <img src="/riko_logo.jpg" style="text-align: center;" width="250">
        <a href="https://facebook.com/RIKODJOFFICIAL/" target="_blank">
			<img src="/fblogo.png" style="float: left; padding-top: 150px;" width="50">
		</a>
    </div>
        <hr>
    <table>
    <tr>
        <h1><b>**ORDER NOW**</b> - GTYM UNLEASHED</h1>
    </tr>
    <tr>
        <img src="/unleashed_front.jpg" style="float: left;" width="50%"> <img src="/unleashed_reverse.jpg" style="float: left;" width="50%">
    </tr>
    <tr>
        <h2>12 Unmixed DJ Friendly Tracks From: Riko, Rob IYF & Defekt</h2>
    <p>
    <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
    <input type="hidden" name="cmd" value="_s-xclick">
    <input type="hidden" name="hosted_button_id" value="Q3AGXTXNKSPH2">
    <input type="hidden" name="on0" value="GTYM UNLEASHED">GTYM UNLEASHED
    <select name="os0">
        <option value="UK RESIDENTS">UK RESIDENTS £15.00 GBP</option>
        <option value="REST OF THE WORLD">REST OF THE WORLD £20.00 GBP</option>
    <input type="hidden" name="currency_code" value="GBP">
    <p>
    <input type="image" src="https://www.paypalobjects.com/en_US/GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online!">
    <img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
    </form>
    </tr>
    <hr>
    <tr>
        <h1>Contagious To The Core (Volume Two) **OUT NOW**</h1>
    </tr>
    <tr>
        <img src="/c2c2front.jpg" style="float: left;" width="50%"> <img src="/c2c2back.jpg" style="float: left;" width="50%">
    </tr>
    <tr>
        <h2> 15 UNMIXED DJ FRIENDLY TRACKS FROM <br/>JOEY RIOT, DJ KURT, RIKO, M-PROJECT, ROB IYF, CLOWNY + MORE.. 🔥<br/><br/>(Shipping Now!)</h2>
    <p>
    <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
    <input type="hidden" name="cmd" value="_s-xclick">
    <input type="hidden" name="hosted_button_id" value="3RHRF7ENP46AU">
                <input type="hidden" name="on0" value="SHIPPING OPTIONS">
            <select name="os0">
                <option value="UK RESIDENTS">UK RESIDENTS £15.00 GBP</option>
                <option value="REST OF THE WORLD">REST OF THE WORLD £20.00 GBP</option>
                <option value="CTTC VOL 1 & 2 BUNDLE">CTTC VOL 1 & 2 BUNDLE £25.99 GBP</option>
            </select>
    <input type="hidden" name="currency_code" value="GBP">
    <p>
    <input type="image" src="https://www.paypalobjects.com/en_US/GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online!">
    <img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
    </form>
    </tr>
    <hr>
	<tr>
		<h1>Contagious Records present:<br/>
		Contagious To The Core (Volume One)</h1>
	</tr>
	<tr>
		<img src="/c2cfront.png" class="album_img"> <img src="/c2cback.png" class="album_img">
	</tr>
	<tr>
		<h2>**OUT NOW!**</br>
		16 UNMIXED DJ FRIENDLY TRACKS FEATURING: JOEY RIOT, KURT, RIKO, M-PROJECT, KUJIN-FU, JONNY EL + MORE!!
		</h2>
		
		<form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">                        
			<select name="os0">
           		        <option value="UK Residents">UK Residents £15.00 GBP</option>
    	     		 	<option value="Rest Of The World">Rest Of The World £20.00 GBP</option>
                        <option value="Contagious To The Core & Step Aside Bundle">Contagious To The Core & Step Aside Bundle £24.99 GBP</option>
               		 </select>
			<p>
				<input type="hidden" name="cmd" value="_s-xclick">
				<input type="hidden" name="hosted_button_id" value="9LB84Z55S2CGN">
			</p>
			<input type="hidden" name="on0" value="Contagious To The Core">Contagious To The Core
			<p>
			<input type="hidden" name="currency_code" value="GBP">
			<input type="image" src="https://www.paypalobjects.com/en_US/GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online!">
			<img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
			</p>
		</form>	
	</tr>
	<hr>
        <tr>
            <h1>Free Download! <br/>
                Contagious Records presents: RIKO POWER<br/>
                Click the album artwork to download!</h1>
        </tr>
        <tr>
            <a href="/rikopower.php"><img src="/riko-power.png" class="album_img"></a>
        </tr>
        <br/>
        <hr>
        <tr>
            <h1>
                Contagious Records Presents: <br/>STEP ASIDE (1XCD)<p>
            </h1>
            <a href="#"><img src="/stepaside_front.jpg" class="album_img"></a>
            <a href="#"><img src="/stepaside_back.jpg" class="album_img"></a>
            <h1>
                **** ALBUM HALF PRICE WHILE STOCK LASTS ****<br/>
                15 Unmixed DJ Friendly Tracks Featuring; <br/>
                KURT, RIKO, M-PROJECT, OUTFORCE, OCTANE, ERUPTION, CLOWNY, BEZZA + MORE...<br/>
                <h2>(Step Aside is also available in digital format, please contact RIKO @ <a href="https://facebook.com/RIKODJOFFICIAL/">facebook.com/RIKODJOFFICIAL</a>)</h2>
            </h1>
                <br/>
                <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_top">
                    <select name="os0">
                        <option value="UK Residents">UK Residents £8.00 GBP</option>
                        <option value="Rest Of The World">Rest Of The World £16.00 GBP</option>
                    </select>
                    <p>
                        <input type="hidden" name="cmd" value="_s-xclick">
                        <input type="hidden" name="hosted_button_id" value="NP6WY2LR75SHQ">
                    </p>
                    <input type="hidden" name="currency_code" value="GBP">
                    <input type="image" src="https://www.paypalobjects.com/en_US/GB/i/btn/btn_buynowCC_LG.gif" border="0" name="submit" alt="PayPal – The safer, easier way to pay online.">
                    <img alt="" border="0" src="https://www.paypalobjects.com/en_GB/i/scr/pixel.gif" width="1" height="1">
                </form>
            <br/>
            <h1>
            Hear previews of the album on <a href="https://soundcloud.com/contagious-records/sets/step-aside" target="_blank">SoundCloud</a><br/>
            <p></p>
            </h1>
        </tr>
        <hr>
        <tr>
            <h1 >
                Contagious Records Presents: <br/> I Am Contagious<p>
            </h1>
            <a href="https://www.trackitdown.net/track/riko/i-am-contagious/hardcore/11505672.html"><img src="/riko-Front.jpg" class="album_img"></a>
            <a href="https://www.trackitdown.net/track/riko/i-am-contagious/hardcore/11505672.html"><img src="/riko-back.jpg" class="album_img"></a>
            <h1>
                Available to order on <a href="https://www.trackitdown.net/track/riko/i-am-contagious/hardcore/11505672.html">TrackItDown</a> now.
            </h1>
        </tr>
        <hr>
        <tr>
            <a href="/hellershardcore.php"><img src="/hhfront.jpg" class="album_img"></a>
            <h1 >Heller's Hardcore OUT NOW!</h1>
            <h3 >Click the album artwork to buy! Only £6 + P&P!!</h3>
        </tr>
        <hr>
        <tr>
            <h1>Free Downloads!</h1>
            <h3>Click the album artwork for downloads.</h3>
        </tr>
        <tr>
            <a href="/album1.php"><img src="/contagiousfront.jpg" class="album_img"></a>
            <a href="/album2.php"><img src="/contagious2front.jpg" class="album_img"></a>
        </tr>
    </table>
</body>
</html>
